code/modules/hydroponics/grown/cocoa_vanilla.dm

```diff
@@ -47,3 +47,41 @@
 	filling_color = "#FEFEFE"
 	tastes = list("vanilla" = 1)
 	distill_reagent = "vanilla" //Takes longer, but you can get even more vanilla from it.
+
+// Bungo Pod
+
+/obj/item/seeds/cocoapod/bungotree
+	name = "pack of bungo tree seeds"
+	desc = "These seeds grow into bungo trees. They appear to be heavy and almost perfectly spherical."
+	icon_state = "seed-bungotree"
+	species = "bungotree"
+	plantname = "Bungo Tree"
+	product = /obj/item/food/grown/bungofruit
+	lifespan = 30
+	maturation = 4
+	yield = 3
+	production = 7
+	genes = list(/datum/plant_gene/trait/repeated_harvest)
```

Comment:
```suggestion

```

you can delete this line, cocoa pods already have repeated harvest, it inherits it